src/SFML/Window/macOS/JoystickImpl.hpp

```diff
@@ -121,7 +121,7 @@ class JoystickImpl
     Joystick::Identification m_identification; ///< Joystick identification
 
     // NOLINTNEXTLINE(readability-identifier-naming)
-    static Location m_locationIDs[sf::Joystick::Count]; ///< Global Joystick register
+    static inline std::array<Location, Joystick::Count> m_locationIDs{}; ///< Global Joystick register
```

Comment:
Adding `inline` here means we can stop defining this static member in the .cpp file. `inline` constants are a great C++17 feature that we've made use of elsewhere.

Reply:
I opened #2952 which makes this change in isolation as well as finding another place where this refactor can take place.